getgist/github.py

```diff
@@ -150,7 +150,7 @@ def list_gists(self):
         """ Prints all gists filenames """       
         for gist in self.get_gists():
             for gist_file in gist.get("files"):
-                print ("[{}] {}".format(gist.get("description"),gist_file.get('filename')))
+                print (f'[{gist.get("description")}]\t{gist_file.get("filename")}\t{gist.get("url")}')
```

Comment:
I am not sure whether tabs would do a great job (imagine we might have long Gist names, form example). Maybe counting chars and padding would be better, but we can fix that later.

More important than that are these two things:

1. `description` might be blank, should we fallback to `id` when that is the case?
2. `url` is the URL of the Gist, not of the file, so we need to build the file URL by appending `#file-name-of-the-file-ext` to the Gist URL (take https://gist.github.com/cuducos/2035fe99e44d9c9a8a1b39d2cc15bc6d#file-getgist-md as an example if you like)